metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -590,10 +590,10 @@ object ServerCommands {
     "Open the Metals logs to troubleshoot issues.",
   )
 
-  val OpenIssue = new OpenBrowserCommand(
-    "https://github.com/scalameta/metals/issues/new/choose",
+  val OpenIssue = new Command(
+    "open-new-github-issue",
```

Comment:
Could you add `OpenFeatureRequest` as a separate one? It should point to `https://github.com/scalameta/metals-feature-requests/issues/new?assignees=&labels=&template=feature-request.yml`